web/src/main/java/com/es/phoneshop/web/controller/pages/CartPageController.java

```diff
@@ -1,25 +1,49 @@
 package com.es.phoneshop.web.controller.pages;
 
 import com.es.core.service.CartService;
+import com.es.phoneshop.web.dataview.UpdateCartItemRequestData;
+import com.es.phoneshop.web.dataview.UpdateCartRequestData;
 import org.springframework.stereotype.Controller;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.ui.Model;
+import org.springframework.ui.ModelMap;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.*;
 
 import javax.annotation.Resource;
+import javax.validation.Valid;
+import java.util.HashMap;
+import java.util.Map;
 
 @Controller
-@RequestMapping(value = "/cart")
 public class CartPageController {
-    @Resource
+    @Resource(name = "httpSessionCartService")
```

Comment:
- use`@Autowired` when you don't assign an explicit name for injected bean (in your case CartService is defined via `@Service`, so you don't have an explicit name for it).
- use `@Resource` otherwise.